src/suffer.cpp

```diff
@@ -78,15 +78,13 @@ static const bionic_id bio_synlungs( "bio_synlungs" );
 
 static const efftype_id effect_adrenaline( "adrenaline" );
 static const efftype_id effect_asthma( "asthma" );
-static const efftype_id effect_bleed( "bleed" );
```

Comment:
Should bleed be removed here?

Reply:
I don't think so. Isn't it used by something else?